src/main/java/gov/usgs/ngwmn/logic/WaterLevelStatistics.java

```diff
@@ -78,14 +81,16 @@ public boolean doesThisMonthQualifyForStats(List<WLSample> monthSamples) {
 	private static final Logger LOGGER = LoggerFactory.getLogger(WaterLevelStatistics.class);
 	
 	public static enum MediationType {
-		BelowLand, 
-		AboveDatum;
+		NONE,
+		BelowLand,
```

Comment:
Is it worth refactoring the BelowLand and AboveDatum enum values to something like BELOW_LAND and ABOVE_DATUM?  It's a convention I've used/seen in other projects.

Reply:
I did not know that. If that is true then I say, yes.